### DISCUSSION TITLE
How can I show the language symbol without the trailing space?

### DISCUSSION THREAD

**Discussion by halloleo** [starship/starship#5748] [Q&A]
Is there a way to 'trim' the language symbol of its obligatory trailing space?

I know I can redefine every symbol without the space, but is there a  more general way, maybe in the format string? Or a way to do it for all situations where no `$version` follows `$symbol`?

_Note:_ I use the **bracketed-segments** preset.

**Comment by matoruru** [comment]
@halloleo 
Yes, you need to redefined symbol in `starship.toml`. For exmample for Rust, the default value of symbol is `'🦀 '`, so if you want it not to have trailing spaces you should re-set it as `'🦀'`.

Reference: https://starship.rs/config/#rust
